crates/orderbook/src/database/orders.rs

```diff
@@ -358,16 +327,16 @@ impl OrderStoring for Postgres {
         order.map(full_order_into_model_order).transpose()
     }
 
-    async fn single_order_with_quote(&self, uid: &OrderUid) -> Result<Option<OrderWithQuote>> {
+    async fn single_order_with_quote(&self, uid: &OrderUid) -> Result<Option<Order>> {
```

Comment:
This now can probably be renamed.
```suggestion
    async fn single_order(&self, uid: &OrderUid) -> Result<Option<Order>> {
```

Reply:
There was `single_order()` function, but used only in tests, so removed it and renamed `single_order_with_quote()` to `single_order()`.